src/guides/v2.3/frontend-dev-guide/layouts/layout-create.md

```diff
@@ -33,6 +33,7 @@ Add the newly created page layout to the `layouts.xml` file of the theme directo
     </layout>
 </page_layouts>
 ```
+> Note that the value of the new layout `id` attribute should match the name of the newly created page-layout XML file.
```

Comment:
Please, use the callout directive for notes. Thank you.

```suggestion
 {:.bs-callout-info}
Note that the value of the new layout `id` attribute should match the name of the newly created page-layout XML file.
```

Reply:
No problem, I was not aware of this directive!